nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestImagingAdapter.java

```diff
@@ -185,5 +185,14 @@ public void testMultiviewPictureAdapter() throws IOException {
         assertNotEquals(0, info.getDepth());
         assertNotNull(info.getColorSpace());
         assertNotNull(info.getFormat());
+
+        // check size info
+        PictureView picView = mvp.getView("Medium");
```

Comment:
For consistency, could've reused `view` instead of instantiating a new variable.